playbooks/build_foreman_proxy_image.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    registry: ''
+    image: foreman-proxy
+    version: latest
+    version_build: 1
+    snippets: []
+    rpms_to_install: []
+    configs_to_copy: []
+  roles:
+    - role: build_foreman_proxy_image
```

Comment:
What is the advantage of the `build_foreman_proxy_image` role that has the same default variables and wraps the set of roles that you would otherwise lay out here?

Reply:
It is easier to test - I can put multiple roles into `molecule/default/converge.yml` which is not possible with playbooks and I would need to create a separate scenarios for each of them.